symplyphysics/laws/kinematic/rotational_inertia/geometries/thin_rod_about_axis_through_center_perpendicular_to_length.py

```diff
@@ -0,0 +1,77 @@
+from sympy import Eq, Symbol as SymSymbol, solve, integrate
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    print_expression,
+    validate_input,
+    validate_output,
+)
+from symplyphysics.core.expr_comparisons import expr_equals
+from symplyphysics.laws.kinematic.rotational_inertia import (
+    rotational_inertia_of_particle as rotational_inertia_def,
+)
+
+# Description
+## A thin rod is rotating about an axis that passes through its center perpendicular to its length.
+
+# Law: I = 1/12 * M * L**2
+## I - rotational inertia of thin rod
+## M - mass of rod
+## L - length of rod
+
+rotational_inertia = Symbol("rotational_inertia", units.mass * units.length**2)
+mass = Symbol("mass", units.mass)
+length = Symbol("length", units.length)
+
+law = Eq(rotational_inertia, mass * length**2 / 12)
+
+
+# Derive law from definition of rotational inertia
+
+# We will consider the thin rod as continuously distributed elements. We can calculate
+# the total rotational inertia of the rod as the sum of rotational inertias of each of
+# its parts, namely, by integrating over the rod's length.
+
+infinitesimal_length = SymSymbol("infinitesimal_length", positive=True)
+infinitesimal_mass = SymSymbol("infinitesimal_mass", positive=True)
+
+# Because the rod is uniform, the ratio of mass to length is equal for all elements of
+# the rod and for the rod itself. This equation basically states that the local linear
+# density of the rod is constant and equal to the ratio of rod's mass to its length.
+
+uniformity_eqn = Eq(infinitesimal_mass / infinitesimal_length, mass / length)
```

Comment:
You could derive it from the constant density condition. But for constant density you could use `rotational_inertia_cartesian_integral` I guess, without integrating manually.